core/src/entities/cyclonedx/models/aggregate_type.rs

```diff
@@ -0,0 +1,39 @@
+use serde::{Deserialize, Serialize};
```

Comment:
I realize there are no comments and that will vomit warnings.  I'm not going to mentions it again and this message was only to let you know that I don't care about them right now because the Culture of Urgency is on fire.